composer.json

```diff
@@ -15,6 +15,7 @@
     "require": {
         "php": "^7.1 || ^8.0",
         "ext-tokenizer": "*",
+        "doctrine/coding-standard": "^6.0 || ^8.1",
```

Comment:
This is in order to be able to run `composer update` on the 7.1 job.

Reply:
Shouldn't it be under `require-dev` instead?

Reply:
Indeed

Reply:
Oh my god yes absolutely :astonished: